ui/app/lenasampler/utils.py

```diff
@@ -128,16 +128,18 @@ def extract_from_audio_file(its_file, audiodir, idprefix, start, duration,
                             abs_ts, outdir):
     audio_file = "%s_%s"%(idprefix, its_file.replace(".its", ".wav"))
     audio_filepath = os.path.join(audiodir, audio_file)
-    outfn = os.path.join(outdir, 
+    # Colon is not allowed in windows paths and causes unhandled error
+    modified_timestamp = str(abs_ts).replace(":", "-")
```

Comment:
Windows gets upset about the colons in this timestamp. Just used a '-' but let me know if you have a better alternative.